go/cmd/dolt/commands/log.go

```diff
@@ -131,8 +154,87 @@ func (cmd LogCmd) logWithLoggerFunc(ctx context.Context, commandStr string, args
 	}
 	if len(opts.tableName) > 0 {
 		return handleErrAndExit(logTableCommits(ctx, dEnv, opts))
+	}*/
+	return logCommits(opts, apr, logRows, queryist, sqlCtx)
+}
+
+// constructInterpolatedDoltLogQuery generates the sql query necessary to call the DOLT_LOG() function.
+// Also interpolates this query to prevent sql injection.
+func constructInterpolatedDoltLogQuery(apr *argparser.ArgParseResults) (string, error) {
+	var params []interface{}
+
+	var buffer bytes.Buffer
+	var first bool
+	first = true
+
+	if apr.Contains(cli.ParentsFlag) && apr.Contains(cli.DecorateFlag) {
```

Comment:
You can simplify this. For example, just alway send the --decorate flag, and always expect that column to exist. There are other examples of this too. Eg, if they don't specify a max number, then we should supply one.